ovirt_imageio/client/_api.py

```diff
@@ -27,11 +27,14 @@
 from .. _internal.handlers import checksum as _checksum
 from .. _internal.nbd import UnixAddress
 
+BUFFER_SIZE = io.BUFFER_SIZE
+MAX_WORKERS = io.MAX_WORKERS
```

Comment:
Why duplicate these constants?

Reply:
This expose the constant in the API. This is exported to the package public names in __init__, and the code is using the _api module to access these values. This way to code
in the client does not have to import the io module.

Reply:
Importing the constants does not make it clear that the names are part of the public module API.

for example:

    from .. _internal import io

Is "io" part of the _api public API?

Practically yes, you can use _api.io, but you should really not do this. This
makes your code fragile and it will break when this import will be replaced with:

    from .. _internal.io import suff

Defining the names in the module makes it clear that this is part of the module
API.

And doing here avoids the ugly internal import in __init__, and make exporting names
from the _api mdoule simpler and more consistent.

Reply:
Got it.

Maybe worth adding a comment (e.g., `# Exposed internal constants`) so that if we expose more internal constants in the public api they get all declared consecutively?

Reply:
Added a comment.